backend/src/controllers/__tests__/CMSLoginController.unit.test.ts

```diff
@@ -0,0 +1,109 @@
+import Server from "../../App";
+import supertest from "supertest";
+import { MongoClient, Db, Collection } from "mongodb";
+import { UNAUTHORISED_TEXT, BAD_REQUEST_TEXT } from "../../constants";
```

Comment:
can we set these constants in jest global? that way we dont have to import in other tests as well.

Reply:
We're testing against the constants the app uses, if we make them global it kinda defeats the point doesn't it? 😕

Reply:
yeah but they're never gonna change. I mean we can import the same constants and put it inside of our jest globals.